src/storage/__tests__/storage.service.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { StorageService } from '../storage.service';
+import { ConfigService } from '@nestjs/config';
+import { MainExceptionFilter } from 'src/app-exception-filters/main-exception.filter';
+import mock = require('mock-fs');
```

Comment:
czemu to tak importujesz?

Reply:
bo import nie działa z tą biblioteką :) mogę spróbować innej, ale ta jest niby najpopularniejsza